Formula/ftjam.rb

```diff
@@ -18,4 +18,23 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    (testpath/"Jamfile").write <<-EOS.undent
+      Main ftjamtest : ftjamtest.c ;
+    EOS
+
+    (testpath/"ftjamtest.c").write <<-EOS.undent
+      #include <stdio.h>
+
+      int main(void)
+      {
+          printf("FtJam Test\\n");
+          return 0;
+      }
+    EOS
+
+    assert_match /Cc ftjamtest.o/, shell_output(bin/"jam").strip
```

Comment:
There are too many of them, so I'd prefer we either fix them en mass or fix only as we update.